apps/api/src/resources/swagger.yaml

```diff
@@ -325,7 +325,7 @@ components:
           format: date-time
           nullable: true
           description: When the application expires. Can be NULL.
-        applicantInformation:
+        contents:
```

Comment:
I changed this for consistency with the database and existing Types but am reviewing if it's needed
If there's preference for `applicantInformation` let me know cc: @surajgoraya